### ISSUE TITLE
Touch:Touch.Tap error iOS

### ISSUE THREAD

**Issue by FelipeFC** [scriptBoris/TouchSam#1]
Touch: Touch.Tap error when "touch: Touch.LongTap" is triggered first.

Error message:

`Object reference not set to an instance of an object`

```
  at TouchSam.iOS.TouchIOS.OnTap (UIKit.UILongPressGestureRecognizer press) [0x000fb] in C:\Users\Scriptboris\source\repos\TouchSam\Library\TouchSam.iOS\TouchIOS.cs:124 
  at UIKit.UILongPressGestureRecognizer+Callback.Activated (UIKit.UILongPressGestureRecognizer sender) [0x00000] in /Library/Frameworks/Xamarin.iOS.framework/Versions/13.20.2.2/src/Xamarin.iOS/UIKit/UIGestureRecognizer.cs:198 
  at (wrapper managed-to-native) UIKit.UIApplication.UIApplicationMain(int,string[],intptr,intptr)
  at UIKit.UIApplication.Main (System.String[] args, System.IntPtr principal, System.IntPtr delegate) [0x00005] in /Library/Frameworks/Xamarin.iOS.framework/Versions/13.20.2.2/src/Xamarin.iOS/UIKit/UIApplication.cs:86 
  at UIKit.UIApplication.Main (System.String[] args, System.String principalClassName, System.String delegateClassName) [0x0000e] in /Library/Frameworks/Xamarin.iOS.framework/Versions/13.20.2.2/src/Xamarin.iOS/UIKit/UIApplication.cs:65 
  at appeduc.iOS.Application.Main (System.String[] args) [0x00002] in D:\Projetos\VisualStudio\AppTest\AppTest\AppTest.iOS\Main.cs:19 
```

Xaml:
```
<Grid Padding="10, 5, 10, 5" 
              HorizontalOptions="FillAndExpand" 
              RowSpacing="1"
              touch:Touch.Color="{x:Static color:JRApp.colorAccentRGB}"
              touch:Touch.Tap="{Binding Source={RelativeSource AncestorType={x:Type viewmodels:NotificacaoGrupoViewModel}}, Path=ItemTappedCommand}"
              touch:Touch.TapParameter="{Binding .}"
              touch:Touch.LongTap="{Binding Source={RelativeSource AncestorType={x:Type viewmodels:NotificacaoGrupoViewModel}}, Path=ItemLongTappedCommand}"
              touch:Touch.LongTapParameter="{Binding .}"
              touch:Touch.LongTapLatency="200">
```

**Comment by Codensconce** [comment]
this plugin only works with StackLayout, I replicated the issue trying to use other Elements.

**Comment by scriptBoris** [comment]
I apologize very much for not answering your question for so long. Sorry one more time.

I fixed this problem in version 1.3.0
https://www.nuget.org/packages/TouchSam/1.3.0

If you wanted to use my package, then it's time to use it :)
